src/schema/type/Annotation.graphql

```diff
@@ -0,0 +1,98 @@
+interface AnnotationTargetInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+}
+
+"An Annotation Target which refers to an external URL"
+type AnnotationURLTarget implements AnnotationTargetInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    url: URL!
+}
+
+"An Annotation Target which refers to a node in the CE"
+type AnnotationCETarget implements AnnotationTargetInterface {
```

Comment:
As discussed in #93. We've gone with this design for now, but it could be changed if needed.
In the case of a number of these types, there is no direct conversion between the type and json-ld. For this reason, I've used the documentation strings to actually add documentation. We can have a discussion about this if you have any suggestions.